backend/sirius-web-spring-collaborative-diagrams/src/main/resources/schema/diagram.graphqls

```diff
@@ -195,6 +202,7 @@ extend type Mutation {
   editLabel(input: EditLabelInput!): EditLabelPayload!
   invokeEdgeToolOnDiagram(input: InvokeEdgeToolOnDiagramInput!): InvokeEdgeToolOnDiagramPayload!
   invokeNodeToolOnDiagram(input: InvokeNodeToolOnDiagramInput!): InvokeNodeToolOnDiagramPayload!
+  invokeDeleteElementToolOnDiagram(input: InvokeDeleteElementToolOnDiagramInput!): InvokeDeleteElementToolOnDiagramPayload!
```

Comment:
use the name `invokeDeleteToolOnDiagram` since element describe another concept of Sirius Components

Reply:
Done